proto/anpr_camera.proto

```diff
@@ -0,0 +1,71 @@
+syntax = "proto3";
+
+package smartcore.bos;
+
+option go_package = "github.com/vanti-dev/sc-bos/pkg/gen";
+
+import "google/protobuf/field_mask.proto";
+import "google/protobuf/timestamp.proto";
+
+
+// AnprCameraApi describes the capability to retrieve detection events from ANPR cameras.
+service AnprCameraApi {
+  rpc GetEvent(GetLastEventRequest) returns (AnprEvent) {}
+  rpc PullEvents(PullEventsRequest) returns (stream PullEventsResponse) {}
+}
+
+// AnprEvent describes a registration plate detection event from an ANPR camera.
+// Includes registration plate information but also vehicle information if the camera supports it.
+message AnprEvent {
+  // The time the detection events occurred.
+  google.protobuf.Timestamp event_time = 1;
+  // The registration plate detected.
+  string registration_plate = 2;
+  // Country of the detected registration plate.
+  string country = 3;
+  // Optional. The area of the detected registration plate. i.e. for UAE this could be Abu Dhabi, Dubai, etc.
+  optional string area = 4;
+  // Optional. The confidence level of the detection as a percentage. If omitted, means unknown confidence level.
+  optional int32 confidence = 5;
```

Comment:
Precent is to use a floating-point type for confidence
https://github.com/smart-core-os/sc-api/blob/5c5bb79965fb358b7ec82475d4f2a0c5404c9288/protobuf/traits/occupancy_sensor.proto#L51
Although I think a double is excessive, a float would be fine here